### PR TITLE
DOC Improve the documentation home page

### PR DESCRIPTION
**Pull request by rth**
Partially addresses https://github.com/pyodide/pyodide/discussions/1954

Add more relevant information to the documentation home page and decrease the depth of the Using pyodide TOC otherwise it generates too many links (particularly for the FAQ)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ governance](https://pyodide.org/en/stable/project/governance.html).
 
 ## Communication
 
+- Blog : [blog.pyodide.org](https://blog.pyodide.org/)
```
Comment (phorward) [inline README.md:76]:
```suggestion
- Blog: [blog.pyodide.org](https://blog.pyodide.org/)
```

File: docs/index.rst [hunk 2]
```diff
@@ -6,19 +6,33 @@ Python with the scientific stack, compiled to WebAssembly.
 Pyodide may be used in any context where you want to run Python inside a web
 browser.
 
-Pyodide brings the Python 3.9 runtime to the browser via WebAssembly, along
-with the Python scientific stack including NumPy, Pandas, Matplotlib, SciPy, and
-scikit-learn. Over 75 packages are currently available. In addition it's
+Pyodide brings the Python 3.9 runtime to the browser via WebAssembly thanks to
```
Comment (phorward) [inline docs/index.rst:9]:
```suggestion
Pyodide brings the Python 3.9 runtime to the browser via WebAssembly, thanks to
```

File: docs/index.rst [hunk 3]
```diff
@@ -6,19 +6,33 @@ Python with the scientific stack, compiled to WebAssembly.
...
+`Emscripten <https://emscripten.org/>`_.
+It builds Python scientific stack including NumPy, Pandas, Matplotlib, SciPy, and
```
Comment (phorward) [inline docs/index.rst:11]:
```suggestion
It builds the Python scientific stack including NumPy, Pandas, Matplotlib, SciPy, and
```

File: docs/index.rst [hunk 4]
```diff
@@ -6,19 +6,33 @@ Python with the scientific stack, compiled to WebAssembly.
...
+scikit-learn. Over 75 packages are currently available. In addition, it's
```
Comment (phorward) [inline docs/index.rst:12]:
```suggestion
scikit-learn. Over 75 packages are currently available. In addition, its
```

File: docs/index.rst [hunk 5]
```diff
@@ -6,19 +6,33 @@ Python with the scientific stack, compiled to WebAssembly.
...
 possible to install pure Python wheels from PyPi.
 
 Pyodide provides transparent conversion of objects between JavaScript and
 Python. When used inside a browser, Python has full access to the Web APIs.
 
+Pyodide development happens on Github: `github.com/pyodide/pyodide <https://github.com/pyodide/pyodide>`_
```
Comment (phorward) [inline docs/index.rst:18]:
```suggestion
Pyodide development happens on GitHub: `github.com/pyodide/pyodide <https://github.com/pyodide/pyodide>`_
```

File: docs/index.rst [hunk 6]
```diff
@@ -65,9 +78,12 @@ information about the project's organization.
...
-Indices and tables
-==================
 
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
+Communication
+-------------
+
+- Blog : `blog.pyodide.org <https://blog.pyodide.org/>`_
```
Comment (phorward) [inline docs/index.rst:85]:
```suggestion
- Blog: `blog.pyodide.org <https://blog.pyodide.org/>`_
```

File: docs/project/about.md [hunk 7]
```diff
@@ -53,6 +53,7 @@ list from there with "The Pyodide development team" like in the example below:
 
 ## Communication
 
+- Blog : [blog.pyodide.org](https://blog.pyodide.org/)
```
Comment (phorward) [inline docs/project/about.md:56]:
```suggestion
- Blog: [blog.pyodide.org](https://blog.pyodide.org/)
```

**Review by phorward — CHANGES_REQUESTED** [review]
Hi @rth thanks for the improving, can you please re-check the suggestions?

File: docs/index.rst (hunk 4 above)
Comment (rth) [reply docs/index.rst:12]: That one was  correct: "it is possible"

**Comment by rth** [comment]
Thanks for the review @phorward !

**Review by phorward — APPROVED** [review]

**Pull request merged**